pkg/controller/extensions/request/extensionrequest.go

```diff
@@ -356,7 +360,7 @@ func (jc *extensionRequestJobCompleter) handleJobCompletion(ctx context.Context,
 			extensionRecord = &v1alpha1.Extension{}
 			n := types.NamespacedName{Name: obj.GetName(), Namespace: obj.GetNamespace()}
 			if err := jc.kube.Get(ctx, n, extensionRecord); err != nil {
-				return errors.Wrapf(err, "failed to retrieve created extension record %s from job %s", obj.GetName(), job.Name)
+				return errors.Wrapf(err, "failed to retrieve created extension record %s in %s from job %s", obj.GetName(), obj.GetNamespace(), job.Name)
```

Comment:
an alternative way of expressing namespace and name is `%s/%s, namespace, name` like is used elsewhere in this package.  not sure if that's consistent across the codebase though :)